src/java/picard/vcf/GenotypeConcordanceCounts.java

```diff
@@ -74,6 +86,76 @@ public void validateCountsAgainstScheme(final GenotypeConcordanceScheme scheme)
     }
 
     /**
+     * Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made
+     * If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.
+     */
+    public double calculateGenotypeConcordance(final GenotypeConcordanceScheme scheme, final boolean missingSitesFlag) {
+        double numerator = 0.0;
+        double denominator = 0.0;
+
+        scheme.validateScheme();
+
+        final TruthState[] allTruthStates = TruthState.values();
+        final CallState[] allCallStates = CallState.values();
+
+        for (final TruthState truthState : allTruthStates) {
+            for (final CallState callState : allCallStates) {
+                if(!missingSitesFlag && isMissing(truthState, callState)) {
+                    continue;
+                }
+                else {
+                    final TruthAndCallStates truthAndCallStates = new TruthAndCallStates(truthState, callState);
+                    final int count = getCount(truthAndCallStates);
+                    if (truthState.ordinal()==callState.ordinal()) {
```

Comment:
My opinion is that this is a dangerous way of doing it.  There's nothing to stop someone from adding another state in one of the enums and throwing off the ordering.  Here's a potentially better way (but @nh13 should weigh in too):

    public enum TruthState {
        MISSING (MISSING_CODE),
        HOM_REF (HOM_REF_CODE),
        HET_REF_VAR1 (HET_REF_CODE),
        etc.;

        private final int code;

        private TruthState(final int code) {
            this.code = code;
        }

        public int getCode() { return code; }
    }

where we have e.g. private final static int MISSING_CODE = 0;